documentation/API_documentation/API Readiness Checklist device-identifier.md

```diff
@@ -0,0 +1,27 @@
+|  2 | Design guidelines from Commonalities applied |   O   |         M         |    M    |    M   |  Y   | 0.4.0                                                                  |
+|  3 | Guidelines from ICM applied                  |   O   |         M         |    M    |    M   |  Y   | 0.2.1                                                                  |
+|  4 | API versioning convention applied            |   M   |         M         |    M    |    M   |  Y   |                                                                        |
+|  5 | API documentation                            |   M   |         M         |    M    |    M   |  Y   | Inline in OAS definition                                               |
+|  6 | User stories                                 |   O   |         O         |    O    |    M   |  Y   | [Device Identifier User Story.md](/documentation/API_documentation/Device%20Identifier%20User%20Story.md) |
+|  7 | Basic API test cases & documentation         |   O   |         M         |    M    |    M   |  Y   | [DeviceIdentifier_retrieve_identifier.feature](/code/Test_definitions/DeviceIdentifier_retrieve_identifier.feature) |
+|  8 | Enhanced API test cases & documentation      |   O   |         O         |    O    |    M   |  N   |                                                                        |
+|  9 | Test result statement                        |   O   |         O         |    O    |    M   |  N   |                                                                        |
+| 10 | API release numbering convention applied     |   M   |         M         |    M    |    M   |  Y   |                                                                        |
+| 11 | Change log updated                           |   M   |         M         |    M    |    M   |  Y   | [CHANGELOG.md](/CHANGELOG.md)                                          |
+| 12 | Previous public release was certified        |   O   |         O         |    O    |    M   |  N   |                                                                        |
+
+To fill the checklist:
+- in the line above the table, replace the api-name, api-version and the rx.y by their actual values for the current API version and release.
+- in the Status column, put "Y" (yes) if the release asset is available or fulfilled in the current release, a "N" (no) or a "tbd". Example use of "tbd" is in case an alpha or release-candidate API version does not yet provide all mandatory assets for the release.
+- in the Comments column, provide the link to the asset once available, and any other relevant comments.
+
+Note: the checklists of a public API version and of its preceding release-candidate API version can be the same.
+
+The documentation for the content of the checklist is here: [API Readiness Checklist](https://wiki.camaraproject.org/display/CAM/API+Release+Process#APIReleaseProcess-APIreadinesschecklist).
```

Comment:
change link to new wiki: https://lf-camaraproject.atlassian.net/wiki/spaces/CAM/pages/14559630/API+Release+Process#API-readiness-checklist